src/tools/genpolicy/src/main.rs

```diff
@@ -34,7 +36,11 @@ mod yaml;
 
 #[tokio::main]
 async fn main() {
+    if env::var("RUST_LOG").is_err() {
```

Comment:
Please confirm with Dan whether we want to introduce this behavior.

Reply:
So the issue here is that warnings are not printed unless `RUST_LOG` is set to at least the `warn` level. So this code sets the env var accordingly, *unless* the user sets it. Thoughts?

Reply:
Users having to enable warnings before they can see the warnings makes sense to me.